src/VisualStudio/CSharp/Impl/ObjectBrowser/DescriptionBuilder.cs

```diff
@@ -6,538 +6,531 @@
-        }
+    protected override async Task BuildTypeDeclarationAsync(
+        INamedTypeSymbol typeSymbol, _VSOBJDESCOPTIONS options, CancellationToken cancellationToken)
+    {
+        BuildTypeModifiers(typeSymbol);
 
-        protected override void BuildTypeDeclaration(INamedTypeSymbol typeSymbol, _VSOBJDESCOPTIONS options)
+        switch (typeSymbol.TypeKind)
         {
-            BuildTypeModifiers(typeSymbol);
+            case TypeKind.Enum:
+                AddText("enum ");
+                break;
 
-            switch (typeSymbol.TypeKind)
-            {
-                case TypeKind.Enum:
-                    AddText("enum ");
-                    break;
-
-                case TypeKind.Struct:
-                    AddText("struct ");
-                    break;
+            case TypeKind.Struct:
+                AddText("struct ");
+                break;
 
-                case TypeKind.Interface:
-                    AddText("interface ");
-                    break;
+            case TypeKind.Interface:
+                AddText("interface ");
+                break;
 
-                case TypeKind.Class:
-                    AddText("class ");
-                    break;
+            case TypeKind.Class:
+                AddText("class ");
+                break;
 
-                default:
-                    Debug.Fail("Invalid type kind encountered: " + typeSymbol.TypeKind.ToString());
-                    break;
-            }
+            default:
+                Debug.Fail("Invalid type kind encountered: " + typeSymbol.TypeKind.ToString());
+                break;
+        }
 
-            var typeNameFormat = new SymbolDisplayFormat(
-                genericsOptions: SymbolDisplayGenericsOptions.IncludeTypeParameters | SymbolDisplayGenericsOptions.IncludeVariance);
+        var typeNameFormat = new SymbolDisplayFormat(
+            genericsOptions: SymbolDisplayGenericsOptions.IncludeTypeParameters | SymbolDisplayGenericsOptions.IncludeVariance);
 
-            AddName(typeSymbol.ToDisplayString(typeNameFormat));
+        AddName(typeSymbol.ToDisplayString(typeNameFormat));
 
-            if (typeSymbol.TypeKind == TypeKind.Enum)
+        if (typeSymbol.TypeKind == TypeKind.Enum)
+        {
+            var underlyingType = typeSymbol.EnumUnderlyingType;
+            if (underlyingType != null)
             {
-                var underlyingType = typeSymbol.EnumUnderlyingType;
-                if (underlyingType != null)
+                if (underlyingType.SpecialType != SpecialType.System_Int32)
                 {
-                    if (underlyingType.SpecialType != SpecialType.System_Int32)
-                    {
-                        AddText(" : ");
-                        AddTypeLink(underlyingType, LinkFlags.None);
-                    }
+                    AddText(" : ");
+                    await AddTypeLinkAsync(underlyingType, LinkFlags.None, cancellationToken).ConfigureAwait(false);
```

Comment:
It looks like this method is trying to stay on the same thread. Should this use `true`?

```suggestion
                    await AddTypeLinkAsync(underlyingType, LinkFlags.None, cancellationToken).ConfigureAwait(true);
```